ipfs/library/Network/IPFS/Client/Files.hs

```diff
@@ -0,0 +1,14 @@
+module Network.IPFS.Client.Files
+  ( API
+  , module Statistics
```

Comment:
Seeing a warning that `module Statistics` exports nothing. It may be because `Network.IPFS.Client.Files.Statistics.Types` is imported qualified. In other places where we re-export, the import isn't qualified.

Reply:
I'll remove it, don't think I need this anymore.